ovirtlib4/vms.py

```diff
@@ -121,6 +138,17 @@ class VmNic(CollectionEntity):
     def __init__(self, *args, **kwargs):
         super(). __init__(*args, **kwargs)
 
+    def get_profile(self):
```

Comment:
This would also make a fine candidate for a property.
Generally speaking I'm forward of creating a property for each class member which is not to be kept privately